src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * added the `form_parent()` function that allows to reliably retrieve the parent form in Twig templates
+ * added the `workflow_build_transition_blocker_list()` function
```

Comment:
`workflow_get_transition_blockers`?

Reply:
in PHP, the method is named `build` (not `get`). I prefer to use the same vocabulary.

Did you see @javiereguiluz suggestion? I like them a lot

Reply:
workflow_transition_blockers ?

Reply:
Let's go with your proposal. I updated the PR. Thanks